### PR TITLE
CODEOWNERS: add Profpatsch to various paths

### PR DESCRIPTION
**Pull request by Profpatsch**
I hope this works.

### CODE REVIEW

**Review by NeQuissimus — APPROVED** [review]

File: .github/CODEOWNERS
Comment (Ericson2314) [inline .github/CODEOWNERS]: Last matching pattern counts, so this should go before my cc-wrapper one

File: .github/CODEOWNERS
Comment (edolstra) [inline .github/CODEOWNERS]: Hm, who made @Ericson2314 owner of stdenv?

**Comment by edolstra** [comment]
I dislike that certain people are now "owner" of various parts of Nixpkgs, without any sort of decision procedure.

File: .github/CODEOWNERS
Comment (vcunat) [reply .github/CODEOWNERS]: Well, I believe our current use of `CODEOWNERS` is mainly for people to express interest in being pinged (automatically) about changes in some files, even though the feature [*can* be used](https://github.com/blog/2392-introducing-code-owners) in much stronger ways.

Comment (edolstra) [reply .github/CODEOWNERS]: Well, the name of the file, and [Github's description](https://help.github.com/articles/about-codeowners/) states it's clearly about ownership.

I didn't know that you could require review from code owners. That might be a good idea to enable.

Comment (vcunat) [reply .github/CODEOWNERS]: Yes, they consider it that way, but so far in nixpkgs there doesn't seem much interest in strict code ownership.  We might explicitly document our intentions at the top of the file, though it's a little risky deviating too much from the semantics envisioned by github.

Comment (NeQuissimus) [reply .github/CODEOWNERS]: I think it would be a decent idea to use "code owners" to replace the mention bot (and I feel that had been mentioned somewhere). Rather than a strict definition of "owning" the code, it would be more in the sense of "I'd like to be invited to review PRs of this code"

Comment (Profpatsch) [reply .github/CODEOWNERS]: Ah, yeah, I see it as replacement of mention bot as well.
I added myself to every file I want to be notified of when it changes.

**Comment by Profpatsch** [comment]
@edolstra I see it as a notification mechanism. Maybe we should add a comment at the top?

File: .github/CODEOWNERS
Comment (Profpatsch) [reply .github/CODEOWNERS]: Ah, sorry. Will fix in a new Pr.

File: .github/CODEOWNERS
Comment (Ericson2314) [reply .github/CODEOWNERS]: All I wanted was notifications. I don't view it as my fief.

Comment (Ericson2314) [reply .github/CODEOWNERS]: @edolstra I'd be worried that if we did use it as an enforcement mechanism, then there'd be no way for non-owners to selectively subscribe to PRs. I want to be able to stay abreast of stdenv changes without needing to also convince you or anyone I should also be a final arbiter of PR approval.

**Pull request merged**
